caliban_toolbox/build.py

```diff
@@ -43,14 +45,15 @@ def compute_cell_size(npz_file, method='median', by_image=True):
             the cell size across the entire npz is returned
 
     Returns:
-        average_sizes: list of typical cell size in NPZ
+        average_sizes: list of typical cell size in NPZ. If no cells, returns None
```

Comment:
```suggestion
        list: list of typical cell size in NPZ. If no cells, returns None.
```

The text before the colon is supposed to be the data type, not the var name.